README.md

```diff
@@ -45,18 +45,35 @@ the serializer generator:
 $ rails g serializer post
 ```
 
-### Support for PORO's and other ORM's.
```

Comment:
i love that you can be _more correct_ about something so tiny :joy: grammar is so cooool